CONTRIBUTING.MD

```diff
@@ -0,0 +1,27 @@
+# Contributing a Devfile Stack to this registry
+
+This document outlines the requirements for contributing a devfile stack to this repository
+
+## Steps
+
+1) Verify your Devfile stack functions with odo.
+  
+    - Core odo functions such as `odo create --devfile <devfile.yaml>`, `odo push`, `odo url create` should work with the devfile.
+    - PR tests on this repository will verify this functionality as well.
+
+2) Add a folder for the stack to `stacks/` in this repository.
+  
+    - Name format should be in the form of `<language>-<framework>`. E.g. `java-quarkus`, `python-django`, etc.
+
+3) Add the devfile.yaml and any other necessary files for the stack under the stack folder.
+
+4) Run the `.ci/build.sh` to build the registry into a container image.
```

Comment:
Is there any setup or system requirement, e.g. platform/software, to run the script successfully? You may want to provide a link here to the setup section in the other doc.

Reply:
Updated.